### PR TITLE
Skelanime + Kanfont Retail OK

### PR DESCRIPTION
**Pull request by hensldm**

### CODE REVIEW

File: src/code/z_skelanime.c [hunk 1]
```diff
@@ -152,6 +152,7 @@ void SkelAnime_DrawFlexLimbLod(PlayState* play, s32 limbIndex, void** skeleton,
             }
             (*mtx)++;
         } else if (limbDList != NULL) {
+            if (1) {}
             MATRIX_TO_MTX(*mtx, "../z_skelanime.c", 954);
```
Comment (fig02) [inline src/code/z_skelanime.c:156]: startin to notice these alot, does the macro itself check if(OOT_DEBUG) 🤔 
something to look into

File: src/code/z_kanfont.c [hunk 2]
```diff
@@ -9,8 +9,9 @@ void func_8006EE50(Font* font, u16 arg1, u16 arg2) {
  * at `codePointIndex`. The value of `character` is the ASCII codepoint subtract ' '/0x20.
  */
 void Font_LoadChar(Font* font, u8 character, u16 codePointIndex) {
+    u8 charTmp = character;
     DMA_REQUEST_SYNC(&font->charTexBuf[codePointIndex],
```
Comment (fig02) [inline src/code/z_kanfont.c:13]: new line

File: src/code/z_skelanime.c [hunk 3]
```diff
@@ -1024,10 +1028,11 @@ void AnimationContext_Update(PlayState* play, AnimationContext* animationCtx) {
         AnimationContext_LoadFrame, AnimationContext_CopyAll,   AnimationContext_Interp,
         AnimationContext_CopyTrue,  AnimationContext_CopyFalse, AnimationContext_MoveActor,
     };
-    AnimationEntry* entry;
+    AnimationEntry* entry = entry = animationCtx->entries;
```
Comment (fig02) [inline src/code/z_skelanime.c:1032]: is this fake or does this make sense? lol

Comment (hensldm) [reply src/code/z_skelanime.c:1032]: Whoops copy paste error. Don't need the second entry.

File: src/code/z_skelanime.c (hunk 1 above)
Comment (cadmic) [reply src/code/z_skelanime.c:156]: I think our graphics macros might be wrong, I've noticed these have been showing up a lot in graphics functions in particular

**Review by fig02 — APPROVED** [review]

**Review by Dragorn421 — APPROVED** [review]

**Review by cadmic — APPROVED** [review]

**Pull request merged**